annotator/src/main/java/org/cbioportal/models/MutationRecord.java

```diff
@@ -0,0 +1,443 @@
+ * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
+ * obligations to provide maintenance, support, updates, enhancements or
+ * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
+ * liable to any party for direct, indirect, special, incidental or
+ * consequential damages, including lost profits, arising out of the use of this
+ * software and its documentation, even if Memorial Sloan-Kettering Cancer
+ * Center has been advised of the possibility of such damage.
+ */
+
+/*
+ * This file is part of cBioPortal CMO-Pipelines.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package org.cbioportal.models;
+
+import java.util.*;
+import org.springframework.util.StringUtils;
+
+/**
+ *
+ * @author Zachary Heins
+ */
+public class MutationRecord {
+    protected String hugoSymbol;
+    protected String entrezGeneId;
+    protected String center;
+    protected String ncbiBuild;
+    protected String chromosome;
+    protected String startPosition;
+    protected String endPosition;
+    protected String strand;
+    protected String variantClassification;
+    protected String variantType;
+    protected String referenceAllele;
+    protected String tumorSeqAllele1;
+    protected String tumorSeqAllele2;
+    protected String dbSnpRs;
+    protected String dbSnpValStatus;
+    protected String tumorSampleBarcode;
+    protected String matchedNormSampleBarcode;
+    protected String matchedNormSeqAllele1;
+    protected String matchedNormSeqAllele2;
+    protected String tumorValidationAllele1;
+    protected String tumorValidationAllele2;
+    protected String matchNormValidationAllele1;
+    protected String matchNormValidationAllele2;
+    protected String verificationStatus;
+    protected String validationStatus;
+    protected String mutationStatus;
+    protected String sequencingPhase;
+    protected String sequencingSource;
+    protected String validationMethod;
+    protected String score;
+    protected String bamFile;
+    protected String sequencer;
+    protected String tumorSampleUUID;
+    protected String matchedNormSampleUUID;
+    protected String tRefCount;
+    protected String tAltCount;
+    protected String nRefCount;
+    protected String nAltCount;
+    protected Map<String, String> additionalProperties = new LinkedHashMap<>();
+
+    public MutationRecord() {}
```

Comment:
Are you ever working with an instance of this class?  Should we add a constructor w/arguments?